library/thrift

```diff
@@ -2,4 +2,5 @@
 
 0.9: git://github.com/ahawkins/docker-thrift@61c3478ab828d3e610f192b442ac2a7221749c47 0.9
 0.9.2: git://github.com/ahawkins/docker-thrift@61c3478ab828d3e610f192b442ac2a7221749c47 0.9
+0.9.3: git://github.com/ahawkins/docker-thrift@23d022c7b84399794349bf4ba7f5568020911f1d 0.9
 latest: git://github.com/ahawkins/docker-thrift@61c3478ab828d3e610f192b442ac2a7221749c47 0.9
```

Comment:
PR is merged!  Ready to update all the commit hashes here and remove `0.9.2`, or would you prefer someone else take over from here? :smile: